app/controllers/declaration/FiscalInformationController.scala

```diff
@@ -63,33 +63,54 @@ class FiscalInformationController @Inject()(
         .fold(
           (formWithErrors: Form[FiscalInformation]) =>
             Future.successful(BadRequest(fiscalInformationPage(itemId, formWithErrors))),
-          validFiscalInformation => updateCacheModelsAndRedirect(itemId, validFiscalInformation)
+          formData =>
+            formData.onwardSupplyRelief match {
+              case FiscalInformation.AllowedFiscalInformationAnswers.yes =>
```

Comment:
Why do we need `updateCacheForYES` and `updateCacheForNo`?
Could you also reformat `YES` to `Yes`, so that it is consistent?

Reply:
The model gets updated differently

Reply:
Done